src/main/java/Domain/User.java

```diff
@@ -3,12 +3,14 @@ public class User {
     private String encoded_password;
     private String username;
     private boolean loggedIn;
+    private boolean isAdmin;
```

Comment:
_isAdmin

Reply:
if you can change all the rest also pls, if not its ok

Reply:
It's not a problem, but why? We agreed that the convention is to use lowercase letters for all fields.